docs/current_docs/quickstart/822194-daggerize.mdx

```diff
@@ -14,8 +14,8 @@ So far, you've learned the basics of calling Dagger Functions. The next step is
 
 Here are the basic steps to Daggerizing an existing project:
 
-1. Initialize a new Dagger Module with `dagger init`
-2. Install other useful Dagger Modules as dependencies with `dagger install`
+1. Initialize a new Dagger module with `dagger init`
+2. Install other useful Dagger modules as dependencies with `dagger install`
```

Comment:
Perhaps this is a good place to link to the daggerverse, otherwise what would someone be installing?

Reply:
Added